ckan/setup/start_ckan_development.sh

```diff
@@ -19,20 +19,40 @@ do
             pip install -r $i/requirements.txt
             echo "Found requirements file in $i"
         fi
+        if [ -f $i/dev-requirements.txt ];
+        then
+            pip install -r $i/dev-requirements.txt
+            echo "Found dev-requirements file in $i"
+        fi
         if [ -f $i/setup.py ];
         then
             cd $i
             python $i/setup.py develop
             echo "Found setup.py file in $i"
             cd $APP_DIR
         fi
+
+        # Point `use` in test.ini to location of `test-core.ini`
+        if [ -f $i/test.ini ];
+        then
+            paster --plugin=ckan config-tool $i/test.ini "use = config:../../src/ckan/test-core.ini"
+        fi
     fi
 done
 
 # Update the plugins setting in the ini file with the values defined in the env var
 echo "Loading the following plugins: $CKAN__PLUGINS"
 paster --plugin=ckan config-tool $CKAN_INI "ckan.plugins = $CKAN__PLUGINS"
 
+# Update test-core.ini DB, SOLR & Redis settings
+echo "Loading test settings into test-core.ini"
+paster --plugin=ckan config-tool $SRC_DIR/ckan/test-core.ini "sqlalchemy.url = postgresql://ckan:ckan@db/ckan_test"
```

Comment:
These databases should be created in the `db` image. And here we should probably use env vars for the connection strings to allow customization

Reply:
Yes, will update this. I forgot to prune the volumes so these were still present in my DB.